tests/test_webgl_no_auto_init_extensions.c

```diff
@@ -0,0 +1,27 @@
+#include <emscripten/html5.h>
+#include <GLES2/gl2.h>
+#include <string.h>
+#include <assert.h>
+
+GL_APICALL void GL_APIENTRY glGenVertexArrays(GLsizei n, GLuint *arrays);
```

Comment:
Looks like this is copied from gl3.h, which I assume you do because this test runs against WebGL1/GLES2. But that brings up the question of why glGenVertexArrays is needed in a GLES2 context; wouldn't real applications on GLES2 use glGenVertexArraysOES?

Reply:
In Emscripten WebGL extensions, all functions are available statically, so `glGenVertexArraysOES` and `glGenVertexArrays` are aliases to each other so it does not matter which name one uses, either in WebGL 1 context app, or a WebGL 2 context app. Updated to use the OES name to be strict.